module/preference/PreferenceDialog.kt

```diff
@@ -329,7 +330,10 @@ object PreferenceDialog {
                 }
             }
             SpacedHorizontalSeparator()
-            preferences.forEach { it.Display() }
+            preferences.forEach {
+                it.Display()
+                ColumnSpacer()
```

Comment:
We didn't have any spacing before and our preferences looked all bunched up, so we provide a spacer after each preference.